.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
```

Comment:
Does OpenShift use these labels?

Reply:
I wonder if we do want them as exceptions? If pinned/security, it does not close? Most of the other Openshift repos do not use github bot. They use openshift bot. For upstream purposes, we would prefer github-bot and maybe we want these exception labels? we can remove them too. Thanks.

Reply:
>I wonder if we do want them as exceptions? If pinned/security, it does not close? 
I certainly believe that we want a way to prevent the bot from marking an issue as stale. If you believe these 2 labels handles reasons OLM would want to prevent the bot from closing the issue, we can mark this as resolved.

> For upstream purposes, we would prefer github-bot and maybe we want these exception labels? we can remove them too. Thanks.
SGTM